### DISCUSSION TITLE
blog/2024/docker-mirrors-cloudflare

### DISCUSSION THREAD

**Discussion by giscus** [yqchilde/yqchilde.github.io#5] [Q&A]
# blog/2024/docker-mirrors-cloudflare

docker hub国内镜像源都下线，在网络上搜集到了利用CloudFlare自建Docker镜像站的办法

https://yqqy.top/blog/2024/docker-mirrors-cloudflare

**Comment by orange886** [comment]
纠正一下获取api TOKEN的模板，应该选择【编辑Cloudflare Workers】模板，并不是选择【Workers Al】，不然无法获取权限。我在github构建的时候老是报错 src/index.js文件中，addEventListener函数下使用了中文字符的逗号，需要修改成英文，但是我修改后，还是一样的报错信息，请问该怎么解决呢？
![报错信息](https://pan.suyanw.cn/view.php/adfab7152b0181a47e560680b509ef2a.png)

**Reply by yqchilde** [reply]
谢谢纠正，获取API TOKEN已更正
此外你的问题我看了下，原本index.js中addEventListener函数使用的就是英文逗号，是你提交时不小心修改了，当你改回英文逗号时，报错已经变成了TOKEN 权限问题，并不是符号问题了，重新创建下token应该就可以了，博客中图片已更新，涉及token读写权限，可以强制刷新网页看一下我的文章中的截图。

<img width="592" alt="Snipaste_2024-06-22_18-59-41" src="https://github.com/yqchilde/yqchilde.github.io/assets/36953759/159d6fa9-50d9-4dd2-9a1f-15f7b36c6781">

<img width="1110" alt="Snipaste_2024-06-22_19-04-34" src="https://github.com/yqchilde/yqchilde.github.io/assets/36953759/6adb49dd-2162-413b-a868-341f1cf3e57e">

**Reply by orange886** [reply]
问题已解决，谢谢解答。

**Comment by zzflybird** [comment]
学习了，已部署 :)
